HuboJointCommand[] joints
int32 num_joints
